boards/arduino/opta/arduino_opta_stm32h747xx_m7_defconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023 Felipe Neves
+# Copyright (c) 2024 DNDG srl
+# SPDX-License-Identifier: Apache-2.0
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable pin controller
+CONFIG_PINCTRL=y
+
+# Use zephyr,code-partition as flash offset
+CONFIG_USE_DT_CODE_PARTITION=y
+
+# Enable correct power supply
+CONFIG_POWER_SUPPLY_SMPS_1V8_SUPPLIES_EXT_AND_LDO=y
+
+# Don't start M4 during the M7 boot (this is what the original Opta does)
+CONFIG_STM32H7_BOOT_M4_AT_INIT=n
+
+# Enable console
+#CONFIG_CONSOLE=y
+#CONFIG_UART_CONSOLE=y
+#CONFIG_UART_LINE_CTRL=y
```

Comment:
console is required

Reply:
The Opta has a single USB that can be used as console if configured as ACM. It can also work as USB storage (I did not yet try this on Zephyr): does it make sense to always enable the console even if the application can configure the USB in a different way?

Reply:
I just pushed a new version that enables the console, but please see my question above.